### DISCUSSION TITLE
Bug Bounty Research Study for Open-Source Maintainers!

### DISCUSSION THREAD

**Discussion by jayala-29** [community/community#102331] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi everyone!

My name is Jessy Ayala, a [PhD student at UC Irvine](https://www.linkedin.com/in/jessy-ayala). Our team has recently submitted a study to a top-tier software engineering conference, and we are curious about the experiences of open-source maintainers who have experience reviewing bug bounty reports.

Your participation will significantly contribute to our research, and we would be grateful for your insights. **If interested, please follow [this link to fill out a brief Likert-scale survey](https://forms.gle/K4YX2HgnUDHJgozh7) which should not take more than a few minutes.**

Thank you for your consideration, your expertise and perspectives are crucial to the success of our study - and to further securing the open-source ecosystem!

**Comment by naveen-98** [comment]
I filled out This forum. Good luck with your research. 🎉

**Reply by jayala-29** [reply]
@naveen-98 Thank you :) we appreciate your contribution!
